### DISCUSSION TITLE
Is it possible to log web browsers?

### DISCUSSION THREAD

**Discussion by wverdi** [amidaware/tacticalrmm#1977] [Q&A]
Newbie question, but i wasnt able to find it anywhere.

Can we log and audit web browsing activity from the endpoints?

**Comment by jmoran-shafer** [comment]
https://discord.com/channels/736478043522072608/1169313753586737184/1180202668019425360

I experimented with a script after this conversation, but it gets real
messy real fast. ChatGPT worked perfectly for getting each browser’s
history in a script.
